cmd/sign-executable/sign-executable.go

```diff
@@ -0,0 +1,291 @@
+package main
+
+import (
+	"archive/zip"
+	"bytes"
+	"context"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+const (
+	getClientSubcommand  = "get-client"
+	signSubcommand       = "sign"
+	notaryClientFilename = "macnotary"
+)
+
+func zipFile(inputFilePath, destinationPath string) error {
+	zipFile, err := os.Create(destinationPath)
+	if err != nil {
+		return errors.Wrap(err, "creating zip file")
+	}
+	defer zipFile.Close()
+
+	zipWriter := zip.NewWriter(zipFile)
+	defer zipWriter.Close()
+
+	inputFile, err := os.Open(inputFilePath)
+	if err != nil {
+		return errors.Wrapf(err, "opening input file '%s'", inputFilePath)
```

Comment:
nit: can we please add "error" (`error opening input file` ) to the beginning of this error and the ones below?

Reply:
I'm actually going to say we shouldn't do this with error messagse. I've been trying to actively stamp out this habit since it produces ugly noise in error messages without adding any value (e.g. "error doing abc: error doing def: failed to do ghi: error doing xyz" is not that readable). I would prefer we converge towards the Go standard library's style of writing error messages or [Uber's style of writing errors](https://github.com/uber-go/guide/blob/master/style.md#errors).

Reply:
TBH I stopped prefacing error messages with "error" because I perceived this to be Kim's preference 🙃 .
[EVG-15160](https://jira.mongodb.org/browse/EVG-15160) should help by establishing what canonical Evergreen code looks like.